conda_smithy/templates/travis.yml.tmpl

```diff
@@ -27,6 +27,9 @@ env:
 {%- endif %}
 {% endblock %}
 
+before install:
+    - brew remove --force $(brew list)
```

Comment:
> > @@ -27,6 +27,9 @@ env:
> >  {%- endif %}
> >  {% endblock %}
> >
> > +before install:
> > +    - brew remove --force $(brew list)
>
> Maybe we should do brew cleanup to free the space taken up by these
packages.

Did you ever run out of space while building on the Travis-CI OS X image?